websites/P/Prime Video/presence.ts

```diff
@@ -72,10 +77,15 @@ presence.on("UpdateData", async () => {
 				presenceData.smallImageText = (await strings).paused;
 				delete presenceData.startTimestamp;
 			} else {
-				const [startTimestamp, endTimestamp] = presence.getTimestamps(
-					video.currentTime,
-					video.duration
-				);
+				const [unformattedCurrentTime, unformattedDuration] = document
+						.querySelector(".atvwebplayersdk-timeindicator-text")
+						.textContent.trim()
+						.split(" / "),
+					[startTimestamp, endTimestamp] = presence.getTimestamps(
```

Comment:
Ditto

Reply:
what do you mean by this? replace the values directly instead of using a variable?